### PR TITLE
Changes to teamPage and index to reflect changes in api

### PR DESCRIPTION
**Pull request by dettloffd**
Changed server/index page to pull from the AWS bucket/API. Also made minor change to teamPage in order to skip first element of the team Json file (which holds team information, not player info).

### CODE REVIEW

**Review by sayemk23 — APPROVED** [review]
nice work

File: index.js [hunk 1]
```diff
@@ -28,7 +28,20 @@ app.get("/", (req, res) => {
 //Team Page
 app.get("/teams/:teamName", (req, res) => {
   const { teamName } = req.params;
-  let team = require("./json_data/" + teamName + ".json");
-  let teamHeader = (teamName.replace("_", " ")).toUpperCase();
-  res.render("teamPage", { team, teamHeader});
+  let teamHeader = teamName.replace("_", " ").toUpperCase();
```
Comment (muhtacinm) [inline index.js]: This line is no longer required as it was used to to get the team name for the title. That information is now stored in the .json files from the API.

File: index.js [hunk 2]
```diff
@@ -28,7 +28,20 @@ app.get("/", (req, res) => {
...
+  let team;
+
+  const getTeam = async () => {
+    try {
+      let response = await axios.get(
+        `https://maqhspyw3j.execute-api.us-east-1.amazonaws.com/dev/${teamName}.json`
+      );
+      team = response.data;
+      res.render("teamPage", { team, teamHeader });
```
Comment (muhtacinm) [inline index.js]: teamHeader is no longer needed here also.

File: views/teamPage.ejs [hunk 3]
```diff
@@ -10,8 +10,8 @@
     <!-- PLAYER CARD -->
     <%# EJS CODE %>
     
-    <div class="container"><% team.forEach(function(player) { %> 
-        <div class="card shadow p-1 mb-4 sm-white rounded" id="playerCard">
+    <div class="container"><% team.slice(1).forEach(function(player) { %>
```
Comment (muhtacinm) [inline views/teamPage.ejs:14]: Nice job on this!

**Review by muhtacinm — CHANGES_REQUESTED** [review]

**Review by muhtacinm — APPROVED** [review]
Nice implementations!

**Review by miqdadhafiz35 — APPROVED** [review]

**Pull request merged**